tests/ui/UnreadIndicatorsTest.js

```diff
@@ -29,17 +29,17 @@ beforeAll(() => {
     // simulate data arriving we will just set it into Onyx directly with Onyx.merge() or Onyx.set() etc.
     global.fetch = TestHelper.getGlobalFetchMock();
 
-    // We need a large timeout here as we are lazy loading React Navigation screens and this test is running against the entire mounted App
-    jest.setTimeout(30000);
     Linking.setInitialURL('https://new.expensify.com/r/1');
     appSetup();
 });
 
-/**
- * @param {RenderAPI} renderedApp
- */
-function scrollUpToRevealNewMessagesBadge(renderedApp) {
-    fireEvent.scroll(renderedApp.queryByA11yLabel('List of chat messages'), {
+beforeEach(() => {
```

Comment:
This was piece was moved from `beforeAll` to `beforeEach` so that if any tests other than the first one take more than 5 seconds they won't time-out.